shopinvader_search_engine/views/product_view.xml

```diff
@@ -42,7 +42,7 @@
     <field name="name">Recompute shopinvader product</field>
     <field name="model_id" ref="product.model_product_template"/>
     <field name="code">
-        for template in object:
+        for template in records:
```

Comment:
IMO this code should no work... `template` is never used

Reply:
Yes thats right, it's luck that my patch fixed the bug ...
I will fix it.